.github/workflows/release_and_update_env.yaml

```diff
@@ -0,0 +1,62 @@
+name: CI/CD
+
+on:
+  pull_request: {}
+  push:
+    branches:
+      - main
+
+concurrency: 
+  group: ${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  py_validate:
+    name: Validate python
+    uses: Energinet-DataHub/.github/.github/workflows/python-pipenv-validate.yaml@helm_releases_action
```

Comment:
I assume all the `@helm_releases_action` references will be updated to `@main` once the `helm_releases_action` branch has been merged into the `.github` repo?